src/Signature/SignatureV4.php

```diff
@@ -82,6 +82,7 @@ public function presign(
         $expires,
         array $options = []
     ) {
+        $expiresTimestamp = $this->convertToTimestamp($expires);
```

Comment:
It does fix the issue of throwing an exception on the calculation, but not the correctness of the relative time cases. For instance, [the tests](https://github.com/aws/aws-sdk-php/blob/master/tests/Rds/AuthTokenGeneratorTest.php) for the [`AuthTokenGenerator`](https://github.com/aws/aws-sdk-php/blob/master/src/Rds/AuthTokenGenerator.php#L58) detect an explicit expires length. Actually, it looks like these are [already failing very infrequently](https://travis-ci.org/aws/aws-sdk-php/jobs/257922253).

We could take this PR as is and supply another fix for relative times, though it'd be preferable to fix both at the same time. The change for this would be using `convertToTimestamp` in `convertExpires` for the non-`strtotime` cases and passing the `$startTimestamp` to `strtotime`'s second param. `strtotime` will properly ignore being given a `$now` parameter when not using a relative time. This would be in lieu of doing a `convertToTimestamp` before sending in to `convertExpires`.